test/data/i3s/plineSonoma3.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This file is added but tests were not changed - is this used?

Reply:
It was something weird... I wasn't aware I changed it, but I changed it back to where it was before.